chain-signatures/node/src/web/mod.rs

```diff
@@ -134,7 +134,7 @@ async fn state(Extension(web): Extension<Arc<AxumState>>) -> Result<Json<StateVi
 
     match &*protocol_state {
         NodeState::Running(state) => {
-            let triple_potential_count = state.triple_manager.len_potential().await;
+            let triple_potential_count = 0; // state.triple_manager.len_potential().await;
```

Comment:
What is this about?

Reply:
whoops, this was handled properly in https://github.com/sig-net/mpc/pull/387 but forgot about it and thought this PR was on top of it already. Will be fixed in the next couple merges